.travis.yml

```diff
@@ -4,10 +4,14 @@ script:
   - sbt "++${TRAVIS_SCALA_VERSION}" "^^${SBT_VERSION}" test scripted
```

Comment:
I can't comment on the jdk line, but maybe you should also test `openjdk8` and `openjdk11`?